test/generic/mommy_recipes.py

```diff
@@ -95,3 +95,14 @@
 )
 
 overrided_save = Recipe('generic.ModelWithOverridedSave')
+
+company = Recipe('generic.Company',
+    name=seq("Doe's Restaurant"),
+)
+position = Recipe('generic.Position',
+    title=seq('CEO'),
+    company=foreign_key(company)
+)
+employee = Recipe('generic.Employee',
+    positions=related(position),
```

Comment:
you should define `name` and `email`, because it is not guaranteed not to have collisions when generating values automatically